wiki/scripts/build.sh

```diff
@@ -34,15 +34,20 @@ rebuild() {
 	echo -e "$(date) $GREEN Updating docs for branch: $1.$RESET"
 	# Generate new docs after merging.
 
+	dir=''
```

Comment:
The version selector did not work when navigating from latest to another version, with a subpath.

e.g. `docs.dgraph.io/query-languages` ----[go to v7.7.1]----> `docs.dgraph.io/v7.7.1`

Fixed in https://github.com/dgraph-io/hugo-docs/pull/16/commits/4428c3a1fb586c9ff91f3465ad2892b620dfe677